File: java/src/main/be.steinsomers.bron_kerbosch/BronKerbosch3_gpx.java
package be.steinsomers.bron_kerbosch;

public final class BronKerbosch3_gpx implements BronKerboschAlgorithm {
    @Override
    public void explore(final UndirectedGraph graph, final CliqueConsumer cliqueConsumer) {
        BronKerboschDegeneracy.explore(graph, cliqueConsumer, PivotChoice.MaxDegreeLocalX);
    }
}

File: kotlin/src/main/be.steinsomers.bron_kerbosch/BronKerbosch3gpx.kt
package be.steinsomers.bron_kerbosch

class BronKerbosch3gpx : BronKerboschAlgorithm {
    override val name: String = "Ver3½-GPX"

    override fun explore(graph: UndirectedGraph, cliqueConsumer: CliqueConsumer) {
        BronKerboschDegeneracy.explore(graph, cliqueConsumer, PivotChoice.MaxDegreeLocalX)
    }
}
